apps/src/templates/teacherHomepage/AnnouncementsCollapsible.jsx

```diff
@@ -18,21 +19,26 @@ const AnnouncementsCollapsible = React.createClass({
 
   render() {
     const { announcements } = this.props;
-    const announcement = announcements[0];
 
     return (
       <CollapsibleSection
         header={i18n.announcements()}
         linkText={i18n.viewAllAnnouncements()}
         link="http://teacherblog.code.org/"
       >
-        <Announcement
-          heading={announcement.heading}
-          description={announcement.description}
-          buttonText={announcement.buttonText}
-          link={announcement.link}
-          image={announcement.image}
-        />
+        <AnnouncementsCarousel>
+          {announcements.length > 0 &&
+            announcements.map((announcement, index) =>
```

Comment:
announcements.length check should be unnecessary, as calling map on an empty array will just return an empty array